Sources/App/Views/Sections/Settings/Server/ServerSettingsViewModel.swift

```diff
@@ -117,6 +127,10 @@ final class ServerSettingsViewModel: ObservableObject {
         ServerConnectionConfiguration(hostname: hostname, port: Int(port) ?? 8080)
       )
 
+      if shouldCreateGitRepository {
```

Comment:
Why not use `isGitRepositoryCreationEnabled`? We will avoid the `shouldCreateGitRepository` parameter

Reply:
👍🏼

Reply:
done 👍